### PR TITLE
Barcode scanners can now be printed.

### PR DESCRIPTION
**Pull request by JohnFulpWillard**
## About The Pull Request

Currently the only way to get a barcode scanner is by spawning as a Curator, this is lame and prevents people to job change into a librarian, so now it can be printed like basically all other service job's tools.

Part of computer tech
image

## Why It's Good For The Game

You no longer have to spawn as a Curator to be able to work in the Library, and Curators can now replace their otherwise completely irreplaceable equipment.

## Changelog

:cl:
qol: The barcode scanner is now part of computer tech and can be printed at the service techfab.
/:cl:

### CODE REVIEW

File: code/modules/research/techweb/all_nodes.dm [hunk 1]
```diff
@@ -1187,6 +1187,7 @@
 		"crewconsole",
 		"idcard",
 		"libraryconsole",
+		"barcode_scanner",
```
Comment (MrMelbert) [inline code/modules/research/techweb/all_nodes.dm:1190]: ABC order (bank machine is screwed up too)

**Comment by ArcaneMusic** [comment]
My mind immediately jumped to "We should add this as another mode to the universal scanner in cargo"

File: code/modules/research/techweb/all_nodes.dm (hunk 1 above)
Comment (JohnFulpWillard) [reply code/modules/research/techweb/all_nodes.dm:1190]: I wanted this to show up next to the library console since they go hand in hand.

**Comment by DrAmazing343** [comment]
> My mind immediately jumped to "We should add this as another mode to the universal scanner in cargo"

Well, it's UNIVERSAL, isn't it? *laugh
For real, though, very good QoL PR. Thank you Willard.

**Pull request merged**